ToolsForHomalg/gap/LazyHLists.gi

```diff
@@ -171,7 +171,7 @@ InstallMethod( \{\},
         
   function( L, l )
     
-    Error( "not implemented yet\n" );
+    return List( l, i -> L[i] );
```

Comment:
This breaks the lazyness, or am I missing something? Is this desirable?

Reply:
You are write, it is better to return `LazyHlist( l, i -> L[i] )` and then evaluating it via `ListOfValues` instead of forcing the evaluation without need.